mcs/class/corlib/Test/System.Threading/icallperf.pl

```diff
@@ -0,0 +1,84 @@
+# Attempt to analyze the performance before and after collecting
```

Comment:
There are hardcoded paths in here.

It's probably worth either:
1. Not checking in this file at all, since it's not runnable.
2. or, fix it up so that it can be used on a clean checkout of Mono (just to gather numbers without comparing).
3. or, leave it non-runnable but with a comment that it's not intended to be working.

Reply:
I changed it to accept command lines to run, and iteration count, but defaults to what I was using.